tests/Unit/Atomic/atomic_xor_global.cpp

```diff
@@ -1,9 +1,9 @@
 // RUN: %cxxamp %s -o %t.out && %t.out
```

Comment:
do we need to clean up the RUN line's cxxamp or leave it as it is?

Reply:
Eventually yes, we'll remove that one.